Source/DafnyLanguageServer/Language/DiagnosticErrorReporter.cs

```diff
@@ -99,12 +99,14 @@ public class DiagnosticErrorReporter : ErrorReporter {
     }
 
     private void AddDiagnosticForFile(Diagnostic item, DocumentUri documentUri) {
```

Comment:
Would be nicer if we can record the errors for each Boogie procedure in a separate non-blocking collection, which is how it works when Dafny is called from the CLI (search for `OutputCollector`), but changing that seems like a lot of hassle, so let's do that as part of bigger Boogie changes :)